src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4251,8 +4511,224 @@ namespace AirflowNetworkBalanceManager {
+			if ( ZonePressure1 <= PressureSet ) {
+				// The highet pressure due to minimum flow rate could not reach Pressure set, bypass pressureset calculation
+				if ( !WarmupFlag ) {
+					if ( ErrCountLowPre == 0 ) {
+						++ErrCountLowPre;
+						ShowWarningError( "The calculated pressure with minimum exhaust fan rate is lower than the pressure setpoint. The pressure control is unable to perform." );
+						ShowContinueErrorTimeStamp( "Calculated pressure = " + RoundSigDigits( ZonePressure1, 2 ) + "[Pa], Pressure setpoint =" + RoundSigDigits( PressureSet, 2 ) );
+					} else {
+						++ErrCountLowPre;
+						ShowRecurringWarningErrorAtEnd( AirflowNetworkNodeData( 3 ).Name + ": The AFN model continues not to perform pressure control due to lower zone pressure...", ErrIndexLowPre, ZonePressure1, ZonePressure1 );
+					}
+				}
+			} else {
+				ExhaustFanMassFlowRate = MaxExhaustMassFlowrate;
+				AIRMOV( );
+				ZonePressure2 = AirflowNetworkNodeSimu( PressureControllerData( 1 ).AFNNodeNum ).PZ;
+				if ( ZonePressure2 >= PressureSet ) {
+					// The lowest pressure due to maximum flow rate is still higher than Pressure set, bypass pressureset calculation
+					if ( !WarmupFlag ) {
+						if ( ErrCountHighPre == 0 ) {
+							++ErrCountHighPre;
+							ShowWarningError( "The calculated pressure with maximum exhaust fan rate is higher than the pressure setpoint. The pressure control is unable to perform." );
+							ShowContinueErrorTimeStamp( "Calculated pressure = " + RoundSigDigits( ZonePressure2, 2 ) + "[Pa], Pressure setpoint = " + RoundSigDigits( PressureSet, 2 ) );
+						} else {
+							++ErrCountHighPre;
+							ShowRecurringWarningErrorAtEnd( AirflowNetworkNodeData( 3 ).Name + ": The AFN model continues not to perform pressure control due to higher zone pressure...", ErrIndexHighPre, ZonePressure2, ZonePressure2 );
+						}
+					}
+				} else {
+					//	if ( ZonePressure1 > PressureSet && ZonePressure2 < PressureSet ) {
+					Par( 1 ) = PressureSet;
+					SolveRegulaFalsi( ErrorToler, MaxIte, SolFla, ExhaustFanMassFlowRate, AFNPressureResidual, MinExhaustMassFlowrate, MaxExhaustMassFlowrate, Par );
+					if ( SolFla == -1 ) {
+						if ( !WarmupFlag ) {
+							if ( ErrCountVar == 0 ) {
+								++ErrCountVar;
+								ShowWarningError( "Iteration limit exceeded pressure setpoint using an exhaust fan. Simulation continues." );
+								ShowContinueErrorTimeStamp( "Exhaust fan flow rate = " + RoundSigDigits( ExhaustFanMassFlowRate, 4 ) );
+							} else {
+								++ErrCountVar;
+								ShowRecurringWarningErrorAtEnd( PressureControllerData( 1 ).Name + "\": Iteration limit warning exceeding pressure setpoint continues...", ErrIndexVar, ExhaustFanMassFlowRate, ExhaustFanMassFlowRate );
+							}
+						}
+					} else if ( SolFla == -2 ) {
+						ShowFatalError( "Zone pressure control failed using an exhaust fan: no solution is reached, for " + PressureControllerData( 1 ).Name );
+					}
+				}
+			}
+		} else {
```

Comment:
Add a comment here to make it clear this block is for relief flow, // PressureCtrlRelief - Pressure control type is Relief Flow ,